scripts/startup/start_panoptes_in_tmux.sh

```diff
@@ -99,6 +99,10 @@ set -x
 # These provide connections between message publishers and subscribers.
 create_and_init_window msg_hub start_messaging_hub.sh
 
+# Create a window running the Google CloudSQL proxy script, connecting
+# to the meta database.
+create_and_init_window meta_db start_cloudsql_proxy.sh meta
```

Comment:
What does this imply about **needing** an internet connection to run POCS?

Reply:
Good point. It actually also implies that you have done the authentication with Google and doesn't handle if not.

More generally, I was thinking about how we could have a "setup" for the startup script to choose which scripts get started. The social messaging, for instance, would be the same. Created #643.